doc/crypto/CTLOG_STORE_new.pod

```diff
@@ -0,0 +1,75 @@
+=pod
+
+=head1 NAME
+
+CTLOG_STORE_new, CTLOG_STORE_free,
+CTLOG_STORE_load_default_file, CTLOG_STORE_load_file -
+Create and populate a Certificate Transparency log list
+
+=head1 SYNOPSIS
+
+ #include <openssl/ct.h>
+
+ CTLOG_STORE *CTLOG_STORE_new(void);
+ void CTLOG_STORE_free(CTLOG_STORE *store);
+
+ int CTLOG_STORE_load_default_file(CTLOG_STORE *store);
+ int CTLOG_STORE_load_file(CTLOG_STORE *store, const char *file);
+
+=head1 DESCRIPTION
+
+A CTLOG_STORE is a container for a list of CTLOGs (Certificate Transparency
+logs). The list can be loaded from one or more files and then searched by LogID
+(see RFC 6962, Section 3.2, for the definition of a LogID).
+
+B<CTLOG_STORE_new>() creates an empty list of CT logs. This is then populated
+by B<CTLOG_STORE_load_default_file>() or B<CTLOG_STORE_load_file>().
+B<CTLOG_STORE_load_default_file>() loads from the default file, which is named
+"ct_log_list.cnf" in the OpenSSL install directory. This can be overridden using
+an environment variable named "CTLOG_FILE".
+B<CTLOG_STORE_load_file>() loads from a caller-specified file path instead.
+Both of these functions append any loaded CT logs to the CTLOG_STORE.
+
+The expected format of the file is:
+
+ enabled_logs=foo,bar
+
+ [foo]
+ description = Log 1
+ key = <base64-encoded public key here>
+
+ [bar]
+ description = Log 2
+ key = <base64-encoded public key here>
+
+Once a CTLOG_STORE is no longer required, it should be passed to
+B<CTLOG_STORE_free>(). This will delete all of the CTLOGs stored within, along
+with the CTLOG_STORE itself.
+
+=head1 NOTES
+
+If there are any invalid CT logs in a file, they are skipped and the remaining
+valid logs will still be added to the CTLOG_STORE. A CT log will be considered
+invalid if it is missing a "key" or "description" field.
+
+=head1 RETURN VALUES
+
+Both B<CTLOG_STORE_load_default_file> and B<CTLOG_STORE_load_file> return 1 if
+all CT logs in the file are successfully parsed and loaded, 0 otherwise.
```

Comment:
This seems quite confusing. Shouldn't there be 3 possible returns:

- Everything loaded ok
- Some entries were invalid but we loaded what we could
- Something really bad happened (e.g. out of memory)

Otherwise how do you distinguish between the latter two results?